app/models/employee.rb

```diff
@@ -8,8 +8,11 @@ class Employee < ApplicationRecord
 
   has_many :given_kudos, class_name: 'Kudo', dependent: :destroy, inverse_of: :employee
   has_many :received_kudos, class_name: 'Kudo', foreign_key: 'receiver_id', dependent: :destroy, inverse_of: :receiver
+  has_many :orders, dependent: :nullify
+  has_many :rewards, through: :orders
 
   def likes
```

Comment:
the name of this method is misleading - I would consider something like `kudos_blance` or `kudos_score`